### PR TITLE
chore: bump ckb-ics to fix a bug

### PR DESCRIPTION
**Pull request by ashuralyk**
come up with https://github.com/synapseweb3/ckb-ics/pull/12, and added new check to ensure SendPacket, WriteAck and ConsumeAck are matched with proper business lock_hash

### CODE REVIEW

File: contracts/ics/base/src/handler.rs [hunk 1]
```diff
@@ -216,6 +220,13 @@ pub fn verify(envelope: Envelope, client: AxonClient) -> CkbResult<()> {
...
             .map_err(Into::into)
         }
-        _ => Err(Error::UnexpectedMsg.into()),
+        MsgType::MsgConsumeAckPacket => {
+            let (old_ibc_packet, old_packet_args) = load_packet_cell(0, Source::Input)?;
+            check_valid_port_id(&old_packet_args.port_id)?;
+
+            let msg: MsgConsumeAckPacket =
+                decode(&envelope.content).map_err(|_| Error::MsgEncoding)?;
+            handle_msg_consume_ack_packet(old_channel, old_channel_args, msg).map_err(Into::into)
```
Comment (blckngm) [inline contracts/ics/base/src/handler.rs]: channel -> packet

File: contracts/ics/base/src/handler.rs [hunk 2]
```diff
@@ -216,6 +220,13 @@ pub fn verify(envelope: Envelope, client: AxonClient) -> CkbResult<()> {
             )
             .map_err(Into::into)
         }
-        _ => Err(Error::UnexpectedMsg.into()),
```
Comment (blckngm) [inline contracts/ics/base/src/handler.rs:219]: Shouldn't remove this line.

Comment (blckngm) [reply contracts/ics/base/src/handler.rs:219]: Still failing because of this?

**Comment by blckngm** [comment]
`navigate_packet` need to handle `MsgConsumeAckPacket`

File: contracts/ics/base/src/handler.rs (hunk 2 above)
Comment (ashuralyk) [reply contracts/ics/base/src/handler.rs:219]: forgot to save file changes

**Comment by ashuralyk** [comment]
@blckngm ConsumeAck packet use default AxonClient to go ahead verify check now

**Comment by blckngm** [comment]
Build is failing.

**Review by blckngm — APPROVED** [review]

**Pull request merged**